Microsoft.Toolkit.Uwp.UI.Controls.DataGrid/DataGrid/DataGridComboBoxColumn.cs

```diff
@@ -336,6 +336,14 @@ protected override FrameworkElement GenerateElement(DataGridCell cell, object da
                 else
                 {
                     textBlockElement.Text = GetDisplayValue(dataItem);
+
+                    if (dataItem is INotifyPropertyChanged notifiableDataItem && (Binding.Mode == BindingMode.OneWay || Binding.Mode == BindingMode.TwoWay))
+                    {
+                        notifiableDataItem.PropertyChanged += (sender, args) =>
```

Comment:
Do we need to worry about unregistering this?  Are there other column types that have similar issues?

Reply:
Perhaps we should using a Binding instead? Just not sure what would be the correct path

Reply:
I don't think other column types would have the same problem as they bind directly to `Binding`. In this case it can't be bound directly to `Binding` as `DisplayMemberPath` points to the property that should be displayed.